application/src/test/resources/standalone/config/router-config.json

```diff
@@ -462,5 +462,9 @@
         "f58761e5-8dd5-4940-a472-872f1236c596"
       ]
     }
-  ]
+  ],
+  "triasApi": {
+    "hideFeedId": true,
+    "hardcodedInputFeedId": "nvbw"
+  }
```

Comment:
This should be put in a separate build-config.json in the sandbox. See how it is done in the emission module.

Reply:
b33f806535203214b22b2f5d057083ad55c6a669